troposphere/static/js/collections/InstancePlaybookCollection.js

```diff
@@ -0,0 +1,20 @@
+import InstanceCollection from "./InstanceCollection";
+import InstancePlaybook from "models/InstancePlaybook";
+import globals from "globals";
+
+export default InstanceCollection.extend({
+    model: InstancePlaybook,
+
+    url: globals.API_V2_ROOT + "/instance_playbooks",
```

Comment:
That gives me an immediate idea of what the data is without knowing anything about the implementation details.